micropython/usb/README.md

```diff
@@ -0,0 +1,85 @@
+# Dynamic USB packages
+
+These packages allow implementing USB functionality on a MicroPython device using pure Python code.
+
+Currently only USB device is implemented, not USB host.
+
+## USB Device support
+
+### Support
+
+USB Device support depends on the low-level [machine.USBDevice](https://docs.micropython.org/en/latest/library/machine.USBDevice.html) class. This class is new and not supported on all ports, so please check the documentation for your MicroPython version. It is possible to implement a USB device using only the low-level USBDevice class. These packages are higher level and easier to use.
```

Comment:
Please wrap lines to approx 80 columns (see eg the lora readme).

Reply:
Suggest "These packages are higher level and easier to use." -> "The packages here are ...".